portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
-* TODO: add some javascript features to portfolio here
-*/
+var slideIndex = 1;
+showSlides(slideIndex);
+
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
```

Comment:
Nit: don't need curly braces for 1-line if-statements.

https://google.github.io/styleguide/jsguide.html#formatting-braces-all